docs/source/receptorctl/receptorctl_status.rst

```diff
@@ -0,0 +1,171 @@
+--------------------------
+Receptorctl status command
+--------------------------
+
+.. contents::
+   :local:
+
+``receptorctl status`` shows the Receptor network status
+
+The command syntax is: ``receptorctl --socket=<socket_path> status [--json]``
+
+``socket_path`` is the control socket address to connect to Receptor.
+   It defaults to Unix socket.
+   Use tcp:// for TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``
+
+``ss --listening --processes --unix 'src = unix:<socket_path>'``
+Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
```

Comment:
Should these three lines go into a code-block?

Reply:
absolutely, great suggestion!